File: C++ Programming/problem on pattern printing/problem48.cpp
/*
48 . Accept number of rows and number of columns from user and display below pattern. 
		Input :  iRow = 5  iCol = 5  
		Output : a b c d e   
					  1 2 3 4 5   
					  a b c d e   
					  1 2 3 4 5   
					  a b c d e  
		
					  
		Author :Milind Nalawade
		
		Date : 02/07/2021

		Language : C++ programming
*/
using namespace std;
#include<iostream>

//Logic

void Print(int row , int col)
{
	char no1='a';
	int no2=1;
	for(int i=0;i<row;i++)
	{	
		no1='a';
		no2=1;
		for(int j=0;j<col;j++)
		{
			if (i%2==0)
			{
				cout<<" "<<no1<<" ";
				no1=no1+1;
			}
			else
			{
				cout<<" "<<no2<<" ";
				no2=no2+1;
			}
		}
		printf("\n");
	}
}

//Entry point function

int main()
{
	int row=0;
	int col=0;
	printf("Enter number of row : ");
	scanf("%d",&row);
	printf("Enter number of column : ");
	scanf("%d",&col);
	Print(row,col);
	return 0;
}		

File: Python Programming/problems on pattern printing/problem48.py
'''
48 . Accept number of rows and number of columns from user and display below pattern. 
		Input :  iRow = 5  iCol = 5  
		Output : a b c d e   
					  1 2 3 4 5   
					  a b c d e   
					  1 2 3 4 5   
					  a b c d e 
					  
		Author :Milind Nalawade
		
		Date : 02/07/2021

		Language : Python 3.9.0
'''

#====
#Logic
#====

def Print(row,col):
	
	for i in range(row):
		no1=0
		no2=1
		for j in range(col):
			if(i%2!=0):
				print(" ",no2,end=" ")
				no2+=1
			else:
				print(" ",chr(ord('a')+no1),end=" ")
				no1+=1
		print()
		
#=============
#Entry point function
#=============
def main():
	row=int(input("Enter number of rows : "))
	col=int(input("Enter number of columns : "))
	Print(row,col)
	
#====
#starter
#====
if __name__=="__main__":
	main()